test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckYourAnswersAndSubmitControllerSpec.scala

```diff
@@ -119,7 +118,7 @@ class CheckYourAnswersAndSubmitControllerSpec
   val completeDutiesSelectedAnswer: DutiesSelectedAnswer                                = sample[DutiesSelectedAnswer]
   val completeCommodityDetailsAnswer: CompleteCommodityDetailsAnswer                    = sample[CompleteCommodityDetailsAnswer]
   val completeNorthernIrelandAnswer: CompleteNorthernIrelandAnswer                      = sample[CompleteNorthernIrelandAnswer]
-  val completeClaimsAnswer: CompleteClaimsAnswer                                        = sample[CompleteClaimsAnswer]
+  val completeClaimsAnswer: ClaimsAnswer                                                = sample[ClaimsAnswer]
```

Comment:
Lets rename completeClaimsAnswer to claimsAnswer

Reply:
Done